shared-helpers/src/locales/tl.json

```diff
@@ -545,7 +545,7 @@
   "errors.householdTooBig": "Napakalaki ng iyong sambahayan.",
   "errors.householdTooSmall": "Napakaliit ng iyong sambahayan.",
   "errors.lastNameError": "Pakilagay ang  Apelyido",
-  "errors.maxLength": "Hindi dapat humigit sa 64 na karakter.",
+  "errors.maxLength": "Hindi dapat humigit sa %{length} na karakter.",
```

Comment:
I'm seeing `Hindi dapat higit sa 64 na character.` with google translate

Reply:
Changed it back to be in sync with core! Good catch 📴